### PR TITLE
Append confirmation link as plain text

### PR DESCRIPTION
**Pull request by tribela**
Some mail application is malfunctioning with links.
link as plaintext is useful when this situation:
screenshot from 2017-09-29 20-05-41
(I was not able to click that link, So I used "view original" feature to complete email confirmation)

### CODE REVIEW

**Comment by Gargron** [comment]
There should be a plaintext version of the e-mail for this, the template ends with text.erb instead of html.haml. I don't think we should put plaintext links into html templates.

**Comment by nightpool** [comment]
@Gargron It's very common to put plain-text confirmation links into emails in case they get blocked by the client. Since getting a squeaky-clean email setup is very, very hard, this seems useful.

File: app/views/user_mailer/confirmation_instructions.en.html.erb [hunk 1]
```diff
@@ -5,8 +5,11 @@
 <p>To confirm your inscription, please click on the following link : <br>
 <%= link_to 'Confirm my account', confirmation_url(@resource, confirmation_token: @token) %>
 
+<p>If above link not worked, Copy and paste this URL</p>
```
Comment (Gargron) [inline app/views/user_mailer/confirmation_instructions.en.html.erb]: > If the above link did not work, copy and paste this URL into your address bar:

**Review by Gargron — CHANGES_REQUESTED** [review]

**Review by Gargron — APPROVED** [review]

**Comment by Kjwon15** [comment]
OMG I changed incorrectly and it merged...
I'm very sorry for this. :cry:

It should be
```html
<p>To confirm your inscription, please click on the following link : <br>
<%= link_to 'Confirm my account', confirmation_url(@resource, confirmation_token: @token) %></p>

<p>If the above link did not work, copy and paste this URL into your address bar: <br>
<span><%= confirmation_url(@resource, confirmation_token: @token) %></span>
```
not
```html
<p>If the above link did not work, copy and paste this URL into your address bar: <br>
<%= link_to 'Confirm my account', confirmation_url(@resource, confirmation_token: @token) %></p>

<p>If above link not worked, Copy and paste this URL</p>
<span><%= confirmation_url(@resource, confirmation_token: @token) %></span>
```

**Comment by Gargron** [comment]
@Kjwon15 Oh, I don't know how I missed it... Tired eyes. Please submit a fix asap...

**Pull request merged**